.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules/
 coverage/
 .twilio-functions
+.DS_Store
```

Comment:
I'd prefer to keep platform specific things out of the project. It's better to [globally ignore `.DS_Store` files](https://code.likeagirl.io/how-to-get-rid-of-ds-store-and-node-modules-in-git-repositories-d37b8a391247) and never worry about them again!

Reply:
TIL ~/.gitignore_global